src/main/resources/staff/templates/document/settlementAgreement.html

```diff
@@ -1 +1,190 @@
-Place holder template
+<html>
+<head>
+  <meta charset="UTF-8"/>
+  <title>{{ claim.referenceNumber }} Settlement agreement</title>
+  <style>
+    @page {
+      size: 8.5in 11in;
+      margin: 10%;
+
+    }
+
+    html {
+      /*noinspection CssNoGenericFontName*/
+      font-family: "Open Sans";
+    }
+
+    body {
+      counter-reset: section;
+    }
+
+    .section {
+      margin-top: 25px;
+    }
+
+    .section .section-heading {
+      margin-bottom: 10px;
+    }
+
+    .section .section-heading strong:before {
+      counter-increment: section;
+      content: counter(section) ". ";
+    }
+
+    table {
+      width: 100%;
+      border-spacing: 0;
+    }
+
+    table td {
+      border-top: 1px solid #999;
+      padding: 5px;
+    }
+    table tr:last-child td {
+      border-bottom: 1px solid #999;
+    }
+
+    table td:first-child {
+      width: 220px;
+      vertical-align: top;
+    }
+
+    table td:last-child {
+      width: auto;
+    }
+
+    header table td, header table tr:last-child td {
+      border: none;
+    }
+
+    header p, header h2 {
+      text-align: right;
+      margin: 0;
+      padding: 0;
+    }
+
+    header td {
+      margin: 0;
+      padding: 0;
+    }
+  </style>
+</head>
+<body>
+
+<header>
+
+  <table>
+    <tr>
+      <td colspan="3"><p>In the County Court Business Centre Online Court</p></td>
+    </tr>
+    <tr>
+      <td colspan="2"><h2><strong>Claim number: </strong></h2></td>
+      <td><h2 style="font-weight: normal"> {{ claim.referenceNumber }}</h2></td>
+    </tr>
+    <tr>
+      <td colspan="2"><p><strong>Date: </strong></p></td>
+      <td><p>{{ settlementReachedAt }}</p></td>
+    </tr>
+  </table>
+</header>
+
+<h2>Settlement agreement</h2>
+
+<div class="section">
+  <div class="section-heading">
+    <strong>Defendant details</strong>
+  </div>
+  <div class="section-body">
+    <table>
+      <tr>
+        <td>Full Name:</td>
+        <td>
+          {{ defendant.fullName }}
+        </td>
+      </tr>
+      <tr>
+        <td>Address:</td>
+        <td>
+          {{ defendant.address.line1 }}
```

Comment:
defendant address is currently the claimant address for some reason, tracking it down